app/boards/arm/waveshare_rp2040_zero/waveshare_rp2040_zero.zmk.yml

```diff
@@ -0,0 +1,12 @@
+identifier: waveshare_rp2040_zero
```

Comment:
This `.zmk.yml` file format is *not* the same as the Zephyr format. See existing ZMK boards for examples of the metadata we use.